### PR TITLE
Kevin Marker

### PR DESCRIPTION
**Pull request by kmarker1101**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -64,6 +79,41 @@ <h4>Island Getaway</h4>
         <div class="btn">Sign Me Up!</div>
       </div>
     </section>
+    <form action="#"><a name="form"></a>
```
Comment (johnoro) [inline index.html]: What is the purpose of the anchor tag with `name=form` on it?

File: less/navigation.less [hunk 2]
```diff
@@ -1 +1,38 @@
...
+    
+            a{
+                padding:10px;
+                text-decoration: none;
+                color:black;
+                font-size:2rem;
+            }
+        }
+    }
+}
```
Comment (johnoro) [inline less/navigation.less]: This file could use a good dose of [prettier](https://prettier.io) for readability.

**Review by johnoro — COMMENTED** [review]
Great work!

File: index.html (hunk 1 above)
Comment (kmarker1101) [reply index.html]: I used it so I could link to it in an href

Comment (johnoro) [reply index.html]: Ah! Usually I'd just put an id on the element (`id="form"`) and then use `href="#form"`. Doing it with an empty `a` tag seems a little unnecessarily confusing!

**Pull request closed without merging**